README.md

```diff
@@ -1 +1,59 @@
-# AwsomeBooksModules
\ No newline at end of file
+# AwsomeBooksModules
+
+Javascript App that keeps track of awesome books
+
+## Built With
+
+-HTML & Javascript
+
+## Getting Started
+
+**Download the files into you repo and you will be good to go.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+\*\*Download the files into you repo and you will be good to go.
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] Let's improve the README by correcting all the typos `your` instead of `you` and also removing all the sections that are not in use in this project such as `Run test`, `Deployment` and `\*\*Download the files into you repo and you will be good to go.` is being repeated. Please add clear instructions on how to install or clone your app to the local machine as this `To get a local copy up and running follow these simple example steps.` does not give a clear instruction.